docs/03-tutorials/00-application-connectivity/ac-03-register-manage-services.md

```diff
@@ -2,275 +2,117 @@
-                     "example": {
-                       "id": "4caad296-e0c5-491e-98ac-0ed118f9474e",
-                       "category": "mammal",
-                       "name": "dog"
-                     },
-                     "properties": {
-                       "id": {
-                         "title": "Id",
-                         "description": "Resource identifier",
-                         "type": "string"
-                       },
-                       "name": {
-                         "title": "Name",
-                         "description": "Pet name",
-                         "type": "string"
-                       },
-                       "category": {
-                         "title": "Category",
-                         "description": "Animal category",
-                         "type": "string"
-                       }
-                     }
-                   }
-                 }
-               }
-             }
-           }
-         }
-       }
-     },
-     "documentation": {
-       "displayName": "Documentation",
-       "description": "Description",
-       "type": "some type",
-       "tags": ["tag1", "tag2"],
-       "docs": [
-           {
-           "title": "Documentation title...",
-           "type": "type",
-           "source": "source"
-           }
-       ]
-     }
-   }'
-   ```
-
-2. Send the request body you prepared in the following call to register a service:
-
-   ```bash
-   curl -X POST -d $REQUEST_BODY https://gateway.$CLUSTER_DOMAIN/$APP_NAME/v1/metadata/services --cert $CLIENT_CERT_FILE_NAME.crt --key $KEY_FILE_NAME.key
-   ```
-
-   A successful response returns the ID of the registered service:
-
-   ```json
-   {"id":"{YOUR_SERVICE_ID}"}
-   ```
+1. Create a Secret that contains your username and password to the external service:
+
+    ```bash
+    kubectl create secret generic $SECRET_NAME --from-literal username=$USER_NAME --from-literal password=$PASSWORD -n kyma-integration
+    ```
+
+2. To register a service with a Basic Authentication-secured API, you must create or modify the respective Application CR. To create an Application CR with the service definition, run this command:
+
+    ```bash
+    cat <<EOF | kubectl apply -f -
+    apiVersion: applicationconnector.kyma-project.io/v1alpha1
+    kind: Application
+    metadata:
+      name: $APP_NAME
+    spec:
+      skipVerify: false
+      services:
+      - id: $TARGET_UUID
+        name: $SERVICE_DISPLAY_NAME
+        displayName: $SERVICE_DISPLAY_NAME
+        description: "Your service"
+        providerDisplayName: "Your organisation"
+        entries:
+        - credentials:
+            secretName: $SECRET_NAME
+            type: Basic
+          targetUrl: $TARGET_URL
+          type: API
+    EOF
+    ```
+
+## Access the registered service 
+
+To check that the service was registered correctly, create a test Pod, and make a call to Application Gateway from within this Pod.   
```

Comment:
```suggestion
To check that the service was registered correctly, create a test Pod and make a call to Application Gateway from within this Pod.   
```

Reply:
This comma is fine here, because they're two separate independent clauses. It's not about Oxford comma. 
See the [**Comma before _and_** section here](https://www.grammarly.com/blog/how-to-use-commas-in-your-writing/).